views/layout.pug

```diff
@@ -15,9 +15,12 @@ html
     div(id='content')
       // include title and copy link button if rendering login or poll
       if (typeof(name) !== 'undefined') 
-        h1(class='ui header' style='font-size: 3em; display: inline-block; margin-right: 10px;')= name
-        div(class='ui icon button' id='copy' data-tooltip='Copy link' style='margin-top: -10px;' data-inverted='')
-          i(class="copy outline icon")
+        h1(id='poll-name' class='ui header' style='font-size: 3em; display: inline-block;')= name
+        div(
+          class='ui icon button' id='copy' style='position: relative; top: -0.5rem; left: 0.5rem;'
```

Comment:
I moved the copy button back to the left cause I think it makes more sense if it's closer to the title :p

Reply:
okay then :p